vision/api/VisionTest/VisionTest.cs

```diff
@@ -118,7 +118,7 @@ public void DetectLandmark()
         {
             var output = Run("landmarks", Path.Combine("data", "tower.jpg"));
             Assert.Equal(0, output.ExitCode);
-            Assert.Contains("Eiffel", output.Stdout);
+            Assert.Matches(@".*(Eiffel|Mars).*", output.Stdout);

```

Comment:
/FYI I have found some of these label results to change, sometimes even to empty, and it still being valid.